logging/src/main/java/tech/pegasys/teku/logging/StatusLogger.java

```diff
@@ -13,84 +13,85 @@
 
 package tech.pegasys.teku.logging;
 
-import org.apache.logging.log4j.Level;
+import static tech.pegasys.teku.logging.ColorConsolePrinter.print;
+
+import com.google.common.primitives.UnsignedLong;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
+import org.apache.tuweni.bytes.Bytes32;
+import tech.pegasys.teku.logging.ColorConsolePrinter.Color;
 
 public class StatusLogger {
 
   public static final StatusLogger STATUS_LOG = new StatusLogger("stdout");
 
-  public enum Color {
-    RED,
-    BLUE,
-    PURPLE,
-    WHITE,
-    GREEN
-  }
+  private final Logger logger;
+  private boolean enabled;
 
-  private static final String resetCode = "\u001B[0m";
+  private StatusLogger(final String name) {
+    this.logger = LogManager.getLogger(name);
+  }
 
-  private final Logger logger;
+  public void setEnabled(final boolean enabled) {
+    this.enabled = enabled;
+  }
 
-  protected StatusLogger(String className) {
-    this.logger = LogManager.getLogger(className);
+  public void genesisEvent(final Bytes32 hashTreeRoot, final Bytes32 genesisBlockRoot) {
+    info("******* Eth2Genesis Event*******", Color.WHITE);
+    info("Initial state root is " + hashTreeRoot.toHexString());
+    info("Genesis block root is " + genesisBlockRoot.toHexString());
   }
 
-  public void log(Level level, String message) {
-    this.logger.log(level, message);
+  public void epochEvent() {
+    info("******* Epoch Event *******", Color.PURPLE);
   }
 
-  public void log(Level level, String message, boolean printEnabled) {
-    if (printEnabled) {
-      this.logger.log(level, message);
-    }
+  public void slotEvent(
+      final UnsignedLong nodeSlot,
+      final UnsignedLong bestSlot,
+      final UnsignedLong justifiedEpoch,
+      final UnsignedLong finalizedEpoch) {
+    info("******* Slot Event *******", Color.WHITE);
+    info("Node slot:                             " + nodeSlot);
+    info("Head block slot:" + "                       " + bestSlot);
+    info("Justified epoch:" + "                       " + justifiedEpoch);
+    info("Finalized epoch:" + "                       " + finalizedEpoch);
```

Comment:
nit: Worth being consistent in whether or not the constant strings have a concatenation break in them so the spacing lines up.
```suggestion
    info("Node slot:                             " + nodeSlot);
    info("Head block slot:                       " + bestSlot);
    info("Justified epoch:                       " + justifiedEpoch);
    info("Finalized epoch:                       " + finalizedEpoch);
```

Reply:
Consistency is good. The `StatusLogger` has yet to reach it's final iteration (hence these kind of inconsistencies) ...but I'll tidy 'em up for this PR 👍